README.md

```diff
@@ -1,66 +1,44 @@
-# landr
+<h1 align="center">
+    Landr
+</h1>
 
-source code = website!
+## 🚀 Quick start
 
-> Build a website for your software projects with one command.
+1. Navigate to the repository
+2. run `GITHUB_TOKEN=[your token] npx landr deploy`
```

Comment:
Can you clarify what permissions the token needs?

Reply:
Great suggestion. Also opened this one https://github.com/balena-io-modules/scrutinizer/issues/56 👍